examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,102 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are stored in previous post processing pulled from an H5 file. The inverse of the T1(p) data is taken to produce the R1(p) data which is fit using the polyfit of the inverse of krho with two degrees of freedom. The R1(p) fit is then fed into the fitting routine for the enhancement data prior to normalization. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+from scipy.io import loadmat
+
+h5_file = "ras.h5"
+nodename = "230706_M67_a"
+# {{{ load data
+Ep = find_file(f"{h5_file}", exp_type="AG_processed_data", expno=f"{nodename}/Ep")
+R1p = find_file(f"{h5_file}", exp_type="AG_processed_data", expno=f"{nodename}/R1p")
+# }}}
+# {{{Find the index where the return powers begin
+flip_idx = np.where(np.diff(Ep.fromaxis("power").data) < 0)[0][0] + 1
+# }}}
+with figlist_var() as fl:
+    # {{{Plot Ep
+    fl.next("Integrated Enhancement")
+    fl.plot(
+        Ep["power", :flip_idx],
+        "o",
+        label="Progressive Saturation Data",
+        capsize=6,
+        alpha=0.5,
+    )
+    fl.plot(
+        Ep["power", flip_idx:],
+        "rx",
+        label="Returning Power Check",
+        capsize=6,
+        alpha=0.5,
+    )
+    # }}}
+    # {{{ plot R1p data
+    fl.next(r"$R_{1}(p)$")
+    fl.plot(R1p, "o", label="Experimental Data")
+    # }}}
+    # {{{load in T100 dataset
+    T10_p = loadmat(
+        search_filename(
+            "T10_DI_water_230412", exp_type="AG_processed_data", unique=True
+        )
+    )["a"][0, :]
+    R10_p = R1p.fromaxis("power").eval_poly(T10_p, "power") ** -1
+    # }}}
+    # {{{ fit krho inverse with two degrees of freedom and then apply to fit R1p
+    krho_inv = Ep.get_prop("acq_params")["concentration"] / (R1p - R10_p)
+    krho_inv_fit = krho_inv.polyfit("power", order=1)
+    krho_inv_fine = R1p.fromaxis("power").eval_poly(krho_inv_fit, "power")
+    R1p_fit = R10_p + Ep.get_prop("acq_params")["concentration"] / krho_inv_fine
```

Comment:
Remember you need to add a script that can add concentration to old data (see comments in main conversation of PR)